<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLinkDialogClass</class>
 <widget class="QDialog" name="EditLinkDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Link</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>0</y>
     <width>245</width>
     <height>85</height>
    </rect>
   </property>
   <property name="title">
    <string>Source</string>
   </property>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>16</y>
      <width>241</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>70</height>
     </size>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>12</y>
       <width>33</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xText">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>8</y>
       <width>49</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="yText">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>32</y>
       <width>49</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>36</y>
       <width>33</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="widthText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>8</y>
       <width>49</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>12</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heightText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>32</y>
       <width>49</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>36</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="localXText">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>12</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="localYText">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>36</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>88</y>
     <width>245</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Destination</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>24</y>
      <width>69</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Level:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nextLevelText">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>20</y>
      <width>165</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>48</y>
      <width>69</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Next X:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nextXText">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>44</y>
      <width>165</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>72</y>
      <width>69</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nextYText">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>68</y>
      <width>165</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>200</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>200</y>
     <width>57</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditLinkDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditLinkDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
